client/src/components/Panels/Common/ToolAdvancedSearch.vue

```diff
@@ -0,0 +1,219 @@
+                </b-alert>
+                <b-alert v-else-if="!itemsLoaded || itemsLoaded.length == 0" class="m-2" variant="danger" show>
+                    No tools found for selected filter(s): {{ filterSettings }}
+                </b-alert>
+                <div v-else>
+                    <b-card v-for="(tool, key) in itemsLoaded" :key="key" no-body>
+                        <b-card-header
+                            v-b-toggle="'accordion-' + key"
+                            :disabled="!hasHelp(tool.help)"
+                            :role="hasHelp(tool.help) ? 'button' : ''">
+                            <span class="pl-2 d-flex justify-content-between">
+                                <span class="row">
+                                    <a
+                                        :href="tool.target === 'galaxy_main' ? 'javascript:void(0)' : tool.link"
+                                        @click.stop="onOpen(tool)">
+                                        <b>{{ tool.name }}</b> {{ tool.description }}
+                                    </a>
+                                    <b-badge class="ml-1">{{ tool.panel_section_name }}</b-badge>
+                                    <b-badge
+                                        v-if="tool.is_workflow_compatible"
+                                        v-b-tooltip.hover
+                                        class="ml-1"
+                                        variant="success"
+                                        title="Can use this tool in Workflows"
+                                        >Workflow</b-badge
+                                    >
+                                    <b-badge v-if="tool.hidden" class="ml-1" variant="danger">Hidden</b-badge>
+                                </span>
+                                <i v-if="hasHelp(tool.help)" class="text-secondary"> Click here to expand tool info </i>
+                            </span>
+                        </b-card-header>
+                        <b-collapse :id="'accordion-' + key" role="tabpanel">
+                            <b-card-body>
+                                <p
+                                    v-if="helpSummary(tool.help)"
+                                    :id="'collapseInfo-' + key"
+                                    class="collapse show"
+                                    v-html="helpSummary(tool.help)"></p>
+                                <a
+                                    v-if="helpSummary(tool.help)"
+                                    data-toggle="collapse"
+                                    :href="'#collapseInfo-' + key"
+                                    role="button"
+                                    aria-expanded="false"
+                                    :aria-controls="'collapseInfo-' + key">
+                                    Show more/less
```

Comment:
Ideally this would show double arrows and could switch from "more" to "less" instead of showing both at the same time.

Reply:
My bad, I changed it to switch from more to less in the upcoming push already